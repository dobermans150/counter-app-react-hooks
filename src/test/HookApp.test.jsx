import React from 'react';
import { shallow } from 'enzyme'
import HookApp from '../HookApp';


describe( 'Pruebas en el archivo HookApp ', () => {

    const wrapper = shallow( <HookApp /> )


    test( 'Debe de retornet un componente', () => {
        expect( wrapper ).toMatchSnapshot()
    } )

} )
