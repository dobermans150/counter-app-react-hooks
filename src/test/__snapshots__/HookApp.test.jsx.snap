// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en el archivo HookApp  Debe de retornet un componente 1`] = `
<div>
  <h1>
    Hola mundo
  </h1>
</div>
`;
